hue/README.md

```diff
@@ -0,0 +1,20 @@
+# HUE - Hadoop User Experience 
+
+This [initialization action] (https://cloud.google.com/dataproc/init-actions) installs the latest version of [HUE](http://gethue.com/) 
+on a master node within a [Google Cloud Dataproc](https://cloud.google.com/dataproc) cluster.
+
+## Using this initialization action
```

Comment:
Could you list which services you think work, and which ones you know to be broken.

I can update the README in relation to other scripts, as I fix them.